lib/python/qmk/cli/find.py

```diff
@@ -11,7 +11,7 @@
     action='append',
     default=[],
     help=  # noqa: `format-python` and `pytest` don't agree here.
-    f"Filter the list of keyboards based on their info.json data. Accepts the formats key=value, function(key), or function(key,value), eg. 'features.rgblight=true'. Valid functions are {filter_help()}. May be passed multiple times; all filters need to match. Value may include wildcards such as '*' and '?'."  # noqa: `format-python` and `pytest` don't agree here.
+    f"Filter the list of keyboards based on their info.json data. Accepts the formats key=value, function(key), or function(key,value), eg. 'features.rgblight=true' or 'length(encoder.rotary, >=20)'. Valid functions are {filter_help()}. May be passed multiple times; all filters need to match. Value may include wildcards such as '*' and '?'." # noqa: `format-python` and `pytest` don't agree here.
```

Comment:
I dont like the special handling here. Imo everything should come from `filter_help` and be self-contained on the classes

Could implement a `get_help()` function on the base class, as simply returning the function's name, and overwrite when needed. Then, for the sake of readability, print them as (instead of the current single-line approach)
```
* One function
* Another function
    Extra docs
 ```
Where the indentation is handled by the class adding a long description and the "common" code looks like `"\n * ".join(klass.get_help() for klass in FilterFunction.__subclasses__())`